### DISCUSSION TITLE
Adding a Keystatic channel to the KeystoneJS Slack

### DISCUSSION THREAD

**Discussion by jschuur** [Thinkmill/keystatic#404] [Ideas]
Could we have #keystatic channel on the existing [KeystoneJS Slack](https://community.keystonejs.com/), since both are from Thinkmill?

It might be a bit of an awkward branding, give the name of the Slack team. I say move both to Discord, but I imagine Thinkmill might use Slack internally, with a number of integrtations.

**Comment by JedWatson** [comment]
Hey @jschuur 👋 

Quietly, one may already exist 🤫

Seriously though, we've talked about this internally, and have a bit of a love/hate relationship with Slack; on the up-side it's great to be able to chat with users (and have a space for people to chat with each other)

On the down-side, all the useful content that gets generated is lost within three months and none of it is public/indexable on Google.

We're trying a new thing and encouraging GitHub for Keystatic for now, mainly to prioritise building up more of a public knowledgebase in discussions here.

Creating a real-time chat space is definitely a time commitment that we see the value in but don't take lightly, when we're ready we're more likely to set up a Discord for this project (and if that goes well, possibly look at moving Keystone over as well)

Right now the focus is building out docs, functionality, and doing support on here.

I'd love to chat more with people who are using Keystatic though, I'll see if we can get something organised.

**Reply by jschuur** [reply]
Makes sense to have a history here in issues and discussions, @JedWatson. Slack is where good support history quickly goes to die.

I had a schema config issue the other day when I got an error screen after I used a dropdown for a `fields.select` item and thought 'maybe I'm just doing this wrong, let me ask in chat real quick', but of course that is exactly the kind of thing you want to hear. Looks like this problem went away with keystatic/core 0.0.108 though.

There was also a small typo in the [Astro docs](https://keystatic.com/docs/installation-astro#keystatic-admin-ui-pages) (`src/pages/eystatic/[...params].astro` is missing a 'k'). I sent a PR for that just now.
